Update FAQ.md (#47)
azat/chdig: Documentation/FAQ.md

  [1]: https://github.com/ClickHouse/ClickHouse/pull/45715
  [2]: https://github.com/ClickHouse/ClickHouse/pull/46480

> ### What is format of the URL accepted by chdiag?

Reviewer: ```suggestion
### What is format of the URL accepted by `chdig`?
```

> example for secure connection with all default connection settings & user name & password (passing the password in the command line is unsafe)

Reviewer: ```suggestion
Example for secure connection with all default connection settings & user name & password (passing the password in the command line is unsafe)
```

```sh
chdig -u 'tcp://user:password@clickhouse-host.com:9440/?secure=true&skip_verify=false&compression=lz4&query_timeout=600s&connection_timeout=5s'
```

> safer option is to pass the password via the ENV

Reviewer: ```suggestion
Safer option is to pass the password via the environment variable:
```
